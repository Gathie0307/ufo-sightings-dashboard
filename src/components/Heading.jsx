const Heading = ({ value }) => {
  return <h1 className='text-center text-3xl font-bold mt-5'>{value}</h1>;
};

export default Heading;
