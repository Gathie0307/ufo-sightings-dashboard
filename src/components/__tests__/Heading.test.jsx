import { render } from '@testing-library/react';
import Heading from '../Heading';

describe('Heading', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Heading value='Hello World' />);

    expect(asFragment()).toMatchSnapshot();
  });
});
