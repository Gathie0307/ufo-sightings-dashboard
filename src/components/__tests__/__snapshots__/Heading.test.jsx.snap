// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Heading > should match snapshot 1`] = `
<DocumentFragment>
  <h1
    class="text-center text-3xl font-bold mt-5"
  >
    Hello World
  </h1>
</DocumentFragment>
`;
